rollup.config.js

```diff
@@ -41,7 +42,7 @@ export default packages.map((pkg) => ({
     typescript({
       tsconfig: `./packages/${pkg}/tsconfig.json`,
     }),
-    json(),
     terser(),
   ],
+  external: ['@walletconnect/modal']
 }))
```

Comment:
bump versions and add changelog, update readme with links to the new examples

Reply:
done